src/The_Cartridge_Header.md

```diff
@@ -127,145 +133,150 @@ Code | Publisher
+`$20` | MBC6
+`$22` | MBC7+SENSOR+RUMBLE+RAM+BATTERY
+`$FC` | POCKET CAMERA
+`$FD` | BANDAI TAMA5
+`$FE` | HuC3
+`$FF` | HuC1+RAM+BATTERY
 
 [^rom_ram]:
-No licensed cartridge makes use of this option. Exact behaviour is unknown.
+No licensed cartridge makes use of this option. The exact behavior is unknown.
 
 [^mbc30]:
-MBC3 with RAM size 64 KByte refers to MBC30, used only in _Pocket Monsters Crystal Version_ for Japan.
-
-## 0148 - ROM Size
-
-Specifies the ROM Size of the cartridge. Typically calculated as "N such that 32 KiB
-<< N".
-
-Code | Size      | Amount of banks
------|-----------|-----------------
- $00 |  32 KByte | 2 (No ROM banking)
- $01 |  64 KByte | 4
- $02 | 128 KByte | 8
- $03 | 256 KByte | 16
- $04 | 512 KByte | 32
- $05 |   1 MByte | 64
- $06 |   2 MByte | 128
- $07 |   4 MByte | 256
- $08 |   8 MByte | 512
- $52 | 1.1 MByte | 72 [^weird_rom_sizes]
- $53 | 1.2 MByte | 80 [^weird_rom_sizes]
- $54 | 1.5 MByte | 96 [^weird_rom_sizes]
+MBC3 with 64 KiB of SRAM refers to MBC30, used only in _Pocket Monsters: Crystal Version_ (the Japanese version of _Pokémon Crystal Version_).
+
+## 0148 – ROM Size
+
+This byte specifies the size of the ROM on the cartridge. In most cases the ROM size
+is given by `32 KiB x (2 ^ <value>)`:
+
+Code  | ROM size  | Number of ROM banks
+------|-----------|-----------------
+`$00` |  32 KiB   | 2 (no banking)
+`$01` |  64 KiB   | 4
+`$02` | 128 KiB   | 8
+`$03` | 256 KiB   | 16
+`$04` | 512 KiB   | 32
+`$05` |   1 MiB   | 64
+`$06` |   2 MiB   | 128
+`$07` |   4 MiB   | 256
+`$08` |   8 MiB   | 512
+`$52` | 1.1 MiB   | 72 [^weird_rom_sizes]
+`$53` | 1.2 MiB   | 80 [^weird_rom_sizes]
+`$54` | 1.5 MiB   | 96 [^weird_rom_sizes]
 
 [^weird_rom_sizes]:
 Only listed in unofficial docs. No cartridges or ROM files using these sizes are known.
-As the other ROM sizes are all powers of 2, these are likely inaccurate. The source for these
+As the other ROM sizes are all powers of 2, these are likely inaccurate. The source of these
 values is unknown.
 
-## 0149 - RAM Size
+## 0149 – RAM Size
+
+This byte indicates how large the RAM chip on the cartridge is, if any.
 
-Specifies the size of the external RAM in the cartridge (if any).
+If the [cartridge type](<#0147 – Cartridge Type>) does not include "RAM" in its name, this should be set to 0.
+This includes MBC2, since its 512 × 4 bits of memory are built directly into the mapper.
 
-Code | Size   | Comment
------|--------|---------
- $00 | 0      | No RAM [^mbc2]
- $01 | -      | Unused [^2kib_sram]
- $02 | 8 KB   | 1 bank
- $03 | 32 KB  | 4 banks of 8 KB each
- $04 | 128 KB | 16 banks of 8 KB each
- $05 | 64 KB  | 8 banks of 8 KB each
+Code  | SRAM size | Comment
+------|----------|---------
+`$00` |   0       | No RAM [^mbc2]
+`$01` |   –       | Unused [^2kib_sram]
+`$02` |   8 KiB   |  1 bank
+`$03` |  32 KiB   |  4 banks of 8 KiB each
+`$04` | 128 KiB   | 16 banks of 8 KiB each
+`$05` |  64 KiB   |  8 banks of 8 KiB each
 
 [^mbc2]:
```

Comment:
This is now covered by the paragraph above, no need to make it a footnote anymore.

```suggestion
`$00` |   0       | No RAM
`$01` |   –       | Unused [^2kib_sram]
`$02` |   8 KiB   |  1 bank
`$03` |  32 KiB   |  4 banks of 8 KiB each
`$04` | 128 KiB   | 16 banks of 8 KiB each
`$05` |  64 KiB   |  8 banks of 8 KiB each

```